translations/en-US.all.yaml

```diff
@@ -0,0 +1,17 @@
+- id: "Update the annotations on a resource"
+  translation: "Update the annotations on a resource"
+
+- id: "watch is only supported on individual resources and resource collections - {{.Count}} resources were found"
+  translation:
+    one: "watch is only supported on individual resources and resource collections - {{.Count}} resources was found"
```

Comment:
How does it know to use the value of {{.Count}} to switch between one and other? and shouldn't it be "1" and other? And is this really the way we want to do plurals?

(Also should be s/resources was/resource was/)